In [1]:
import thinkstats2
import thinkplot
import nsfg
import math
import random
#import brfss
import scipy.stats

In [2]:
# df = brfss.ReadBrfss()
# df.columns.tolist()
# oops -- realized that we are given the stats for the population

In [3]:
mu = 178
sigma = 7.7
dist = scipy.stats.norm(loc=mu, scale=sigma)
dist.mean(), dist.std()

(178.0, 7.7000000000000002)

In [4]:
# cdf for height = 6'1" (73 inches) - cdf for 5'10" (70 inches)
cm_per_inch = 2.54
six_one_to_cm = 73*cm_per_inch
five_ten_to_cm = 70*cm_per_inch
print(six_one_to_cm)
print(five_ten_to_cm)


185.42000000000002
177.8


In [5]:
percentage = ( dist.cdf(six_one_to_cm) - dist.cdf(five_ten_to_cm) ) * 100
percentage

34.274683763147458

* 34.3% seems reasonable, since it's a part of the normal distribution right around the mean